js/witness_calculator.js

```diff
@@ -119,15 +150,13 @@ class WitnessCalculator {
         this.i32 = new Uint32Array(memory.buffer);
         this.instance = instance;
 
-        this.n32 = (this.instance.exports.getFrLen() >> 2) - 2;
-        const pRawPrime = this.instance.exports.getPRawPrime();
-
+        this.n32 = this.instance.exports.getFieldNumLen32();
+        this.instance.exports.getRawPrime();
```

Comment:
This breaks circom 1 compatibility, it just errors since `getFieldNumLen32` doesn't work with circom 1 exports. I'm guessing that means this needs a semantic versioning bump.

Reply:
@nalinbhardwaj can we look for the export and differentiate the implementation based to support both?

Reply:
Good idea, will test that.

Reply:
@nalinbhardwaj Any updates on this? Was talking to Jordi about this PR and he gave me the 👍 that this should support both circom1 and circom2

Reply:
Just added the commit that adds support for both circom 1 and 2: circom 1 circuits work perfectly now but seems like in the process I introduced some regression for circom 2 circuits and they seem to be breaking again, just deadlocking somewhere and freezing up my browser currently.

Reply:
This is resolved now.